scripts/xdcc_autoget.pl

```diff
@@ -200,6 +203,59 @@ sub ag_getfinished		#reads in finished packs list
 	chomp(@finished);
 	@finished = ag_uniq(@finished);
 	close(FINISHED);
+	&ag_clearcache;
+	@finished = &ag_uniq(@finished);
```

Comment:
do you need to run ag_uniq  2 times here :-)? also please without `&` (& = perl4 style)